spec

```diff
@@ -744,12 +744,23 @@ beginseg
     include "$(BUILD_DIR)/src/libu64/pad.o"
     include "$(BUILD_DIR)/src/libu64/code_800FC620.o"
     include "$(BUILD_DIR)/src/libu64/padsetup.o"
+#endif
+#if PLATFORM_N64
+    include "$(BUILD_DIR)/src/libu64/gfxprint.o"
```

Comment:
there's probably a faulty merge around here, e.g. gfxprint.o is included twice under PLATFORM_N64

Reply:
Its not a faulty merge, this is what cadmic recommended I put here (as a solution to the mtxuty-cvt problem)
I do agree it seems weird, but it seems to match? I dont understand how

Reply:
I'm guessing the linker is smart enough to not actually include the same .o twice
where was the discussion on this?

Reply:
oh I didnt know the linker would do that. The problem was just with my copy pasting, I see the block that needs to be removed